components/brave_wallet/rust/lib.rs

```diff
@@ -2,6 +2,7 @@ use core::fmt;
 use ed25519_dalek_bip32::derivation_path::{
     ChildIndexError, DerivationPath, DerivationPathParseError,
 };
+use ed25519_dalek_bip32::ed25519_dalek::ed25519::signature::Signature as SignatureTrait;
```

Comment:
This line is unnecessary in my build?

Reply:
It is necessary in order to use trait function `from_bytes`

Reply:
Oh, I see, we need this until Cargo.lock gets updated?

Reply:
I'm currently bumping the version. And I will remove it in the upcoming commit.

Reply:
fixed in 2eb4005416a8733e9ee0dc6ff6a96284dc948bcf